src/settingsScreen.js

```diff
@@ -364,10 +364,11 @@ class SettingsScreen {
         }
     }
     /**
-     * @param {object} { maxBasalRate string, maxBolus string }
-     * @example await settings.SetDeliveryLimits({maxBasalRate:'1.0', maxBolus:'10.0'})
+     * @param {object} limits { maxBasalRate string, maxBolus string }
+     * @param {function} postChecksFn function that is called after the setting is set
+     * @example await settings.SetDeliveryLimits({maxBasalRate:'1.0', maxBolus:'10.0'}, checks)
      */
-    async SetDeliveryLimits(limits) {
+    async SetDeliveryLimits(limits, postChecksFn) {
```

Comment:
[nit] I would rename `postChecksFn` to `additionalExpectations`.

Reply:
good idea